src/config/model.rs

```diff
@@ -367,16 +353,60 @@ impl Config {
     /// This is a little helper-function like which uses the the name and email
     /// of the account to create a valid address for the header of the envelope
     /// of an msg.
+    ///
+    /// # Hint
+    /// If the name includes some special characters like a whitespace, comma or semicolon, then
+    /// the name will be automatically wrapped between two `"`.
+    /// 
+    /// # Exapmle
+    /// ```
+    /// use himalaya::config::model::{Account, Config};
+    /// 
+    /// fn main() {
+    ///     let config = Config::default();
+    ///
+    ///     let normal_account = Account::new(Some("Acc1"), "acc1@mail.com");
+    ///     // notice the semicolon in the name!
+    ///     let special_account = Account::new(Some("TL;DR"), "acc2@mail.com");
+    ///
+    ///     // -- Expeced outputs --
+    ///     let expected_normal = Account {
+    ///         name: Some("Acc1".to_string()),
+    ///         email: "acc1@mail.com".to_string(),
+    ///         .. Account::default()
+    ///     };
+    ///
+    ///     let expected_special = Account {
+    ///         name: Some("\"TL;DR\"".to_string()),
+    ///         email: "acc2@mail.com".to_string(),
+    ///         .. Account::default()
+    ///     };
+    ///
+    ///     assert_eq!(config.address(&normal_account), "Acc1 <acc1@mail.com>");
+    ///     assert_eq!(config.address(&special_account), "\"TL;DR\" <acc2@mail.com>");
+    /// }
+    /// ```
     pub fn address(&self, account: &Account) -> String {
         let name = account.name.as_ref().unwrap_or(&self.name);
+
+        let special_chars: Vec<&str> = name
```

Comment:
I think the algorithm can be improved here. We don't need to collect _all_ special chars, we just need to know if there is _at least_ one. What about sth like `let has_special_chars = name.contains(|c| "()<>[]:;@.,".contains(c));`?